values.yaml

```diff
@@ -634,6 +634,26 @@ keycloak:
   extraVolumeMounts: |
     - name: incore-theme
       mountPath: /opt/bitnami/keycloak/themes/incore-theme
+    
+
+# ----------------------------------------------------------------------
+# APPROVAL APP CONFIGURATION
+# ----------------------------------------------------------------------
+approval:
+  replicas: 1
+  image:
+    repository: approval-app
+    tag: latest
+  service:
+    type: ClusterIP
+    port: 5000
+  ingress:
+    annotations:
+  ADMIN_LIST: jonglee1@illinois.edu,cmnavarr@illinois.edu
```

Comment:
let's not put the real emails in the repository? We can put this in values-incore-*.yaml

Reply:
Removed and updated in this PR https://git.ncsa.illinois.edu/kubernetes/clusters/incore-dev/-/merge_requests/13/diffs

I don't think we will use values overwrite anymore moving on? hence I left that out.